packages/venia-ui/lib/components/LinkButton/linkButton.css

```diff
@@ -6,7 +6,6 @@
     text-decoration: underline;
     line-height: 1.25rem;
     max-width: 100%;
-    outline: none;
```

Comment:
@davemacaulay was this set to `none` for a reason in the Buttons PR?

Reply:
I think it was incorrectly inherited, so fine to remove here.